File: src/nikeno/Tenki/HelpActivity.java
package nikeno.Tenki;

import nikeno.Tenki.R;
import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebView;

public class HelpActivity extends Activity {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.help);
		
		WebView webView = (WebView)findViewById(R.id.webView); 
		webView.loadUrl("file:///android_asset/help.html");
	}  
}

File: app/src/main/java/nikeno/Tenki/activity/HelpActivity.kt
package nikeno.Tenki.activity;

import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebView;

import nikeno.Tenki.R;

public class HelpActivity extends Activity {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.help);
		
		WebView webView = findViewById(R.id.webView);
		webView.loadUrl("file:///android_asset/help.html");
	}  
}
